mobify/documentation-theme
# WEB-2067 Reduce NPM vulnerabilities

There were some NPM vulnerabilities shown in the SDK project which was then traced back to here. Thus, some dependencies were added and/or updated.

 **JIRA**: https://mobify.atlassian.net/secure/RapidBoard.jspa?rapidView=271&projectKey=WEB&modal=detail&selectedIssue=WEB-2067

## Changes
- Pulled upstream from https://github.com/sintaxi/harp
- Address some other NPM vulnerabilities with `npm audit` and following the prompts 

## How to test-drive this PR
- Change npm version to 6.0.0 via `npm install -g npm@6.0.0`
- Run `npm audit`
- See that there are the following vulnerabilities:
[!] found 2 vulnerabilities (1 low, 1 moderate) in 4739 scanned packages
